src/com/lftechnology/java/training/dipak/employeemanagement/ui/EmployeeView.java

```diff
@@ -0,0 +1,242 @@
+                LOGGER.info("\nPlease re-enter the values.");
+            }
+
+        }
+
+        employee.setAddress(address);
+        employee.setDepartment(department);
+        employee.setFullName(fullName);
+        employee.setIsTerminated(isTerminated);
+        employee.setUserName(userName);
+        employee.setPassword(password);
+        employee.setRole(role);
+
+        EmployeeController ec = new EmployeeController();
+        count = ec.addEmployee(employee);
+
+        if (count == 0) {
+            LOGGER.info("\n\t\tInsertion failed. No users were added. Please check and enter the data again.");
+        } else {
+            LOGGER.log(Level.INFO, "\nNumber of users added={0}", count);
+        }
+
+    }
+
+    /**
+     * <p>
+     * This method is used to take up the full name from the console who are to be deleted. And after completion of that it calls the
+     * terminateEmployee method placed on the controller layer.
+     * </p>
+     * 
+     * @param em
+     * @param sc
+     */
+    public void terminateEmployee(Employee em, Scanner sc) {
+
+        String fullName = null;
+        int count = 0;
+        try {
+            LOGGER.info("\nPlease enter the employee fullname::");
+            fullName = sc.nextLine();
+            em.setFullName(fullName);
+
+            EmployeeController ec = new EmployeeController();
+
+            ec.terminateEmployee(em);
+
+        } catch (Exception ex) {
+            LOGGER.log(Level.INFO, exceptionOccurred, ex);
+        }
+        if (count == 0) {
+            LOGGER.info("\n\tNo users were deleted. Please check the fullname again.");
+        } else {
+            LOGGER.log(Level.INFO, "\nNumber of users deleted = {0}", count);
+        }
+    }
+
+    /**
+     * <p>
+     * This method takes up the input from the console for the various filters that are used to search the employee data. After completion
+     * of that,it calls the viewEmployee method in the controller layer.
+     * </p>
+     * 
+     * @param empl
+     * @param sc
+     */
+    public void viewEmployee(Employee empl, Scanner sc) {
+        ResultSet rs = null;
+        String fullName = null;
+        String department = null;
+        String address = null;
+        try {
+            LOGGER.info("\nPlease enter the employee fullname::");
+            fullName = sc.nextLine();
+
+            LOGGER.info("\nPlease enter the department::");
+            department = sc.nextLine();
+
+            LOGGER.info("\nPlease enter the address::");
+            address = sc.nextLine();
+
+            empl.setFullName(fullName);
+            empl.setAddress(address);
+            empl.setDepartment(department);
+
+            EmployeeController ec = new EmployeeController();
+
+            rs = ec.viewEmployee(empl);
+
+            try {
+                if (rs.next()) {
+                    LOGGER.info("\n eid \t\t username \t\t fullname \t\t department \t\t address \t\t\n");
+                    do {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS134)